### DISCUSSION TITLE
[FEATURE REQ] Create a mempool explorer guide

### DISCUSSION THREAD

**Discussion by federicociro** [minibolt-guide/minibolt#83] [Ideas]
### What & why

> As a Minibolt user, I want to include a Bitcoin explorer as Mempool.
The benefits of a mempool explorer include the ability to view the current number of unconfirmed transactions, the fees associated with those transactions, and the amount of time that each transaction has been waiting to be confirmed. This information can be useful for bitcoin users and miners, as it allows them to make informed decisions about transaction fees and confirmation times. Additionally, mempool explorers can be used to detect and track network congestion, which can be helpful for identifying and addressing scalability issues on the bitcoin network.

### How

I am thinking of creating a guide on how to install a Bitcoin Mempool explorer. This guide would explain the steps required to set up and configure a mempool explorer, including any software or hardware requirements, as well as any necessary configurations.

### Additional context

Mempool will be connected with Bitcoin Core through **.cookie file** and not RPC credentials
I can help with this once PR 2846 is merged into Mempool repository.
[Mempool PR 2846 by @antonilol](https://github.com/mempool/mempool/pull/2846)

**Comment by twofaktor** [comment]
I waiting for this PR to be approved: https://github.com/mempool/mempool/pull/2846

It seems that the developer that starts to move 🔥 

When it's merged into the master branch, I'll create the PR! I will be attentive to the movements 👀

**Reply by Filouman** [reply]
Looks like it has now been merged on November 10

**Reply by d6n13l0l1v3r** [reply]
but don't have release yet.

**Reply by twofaktor** [reply]
![imagen](https://github.com/minibolt-guide/minibolt/assets/89636253/2dff574d-7445-455a-89ec-4ba7f91e72fb)

**Comment by twofaktor** [comment]
Build in progress at https://github.com/minibolt-guide/minibolt/pull/128, conversation moved to the PR.
